defaults/main.yml

```diff
@@ -112,6 +112,15 @@ telemetry_tls_privatekey:
 telemetry_tls_certificate:
 telemetry_tls_ca_certificate:
 
+# JWT configuration
+jwt_remote_certificate_path: /etc/hub/jwt
```

Comment:
variable naming should be consistent with the one for telemetry - either both include the `remote` or both don't. Also, it's not a certificate, it's just a private and public key.